package.json

```diff
@@ -5,13 +5,18 @@
   "license": "Apache-2.0",
   "author": "Holepunch",
   "main": "index.js",
+  "files": [
+    "index.js",
+    "bare-noop"
+  ],
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "standard"
   },
   "dependencies": {
     "bare-assert": "^1.0.2",
     "bare-fs": "^4.1.2",
+    "bare-noop": "file://./bare-noop",
```

Comment:
This is bound to create package hoisting issues and odd behaviour in non-`npm` package managers. Adding the `bare-module` import mapping alone is sufficient.